public/index.css

```diff
@@ -18,7 +18,7 @@
     width: 500px;
     border: black 1px solid;
     border-radius: 5%;
-    background-image: url('public/image/dish-image.jpeg'); 
+    background-image: url('./image/dish-image.jpeg'); 
```

Comment:
Altered the file path code in the CSS. It wasn't working so i just deleted and tyoed,  "." then "/" and selected the public folder and then the image. VS code then filled out the code for me.